test/integration/users.test.js

```diff
@@ -0,0 +1,156 @@
+const supertest = require('supertest');
+const { factory } = require('factory-girl');
+
+const dbHelper = require('../helpers/db');
+const usersTestData = require('../data/users');
+const app = require('../../app');
+const { errorMessages } = require('../../app/constants');
+
+require('../setup');
+
+const postUsersPath = '/users';
+const defaultHeaders = ['Accept', 'application/json'];
+
+describe('Create users POST  /users', () => {
+  describe('Should work correctly with satisfactory case', () => {
+    let response = {};
+
+    beforeEach(async done => {
+      response = await supertest(app)
+        .post(postUsersPath)
+        .set(...defaultHeaders)
+        .send(usersTestData.createUserRequestBody);
+
+      done();
```

Comment:
This done() is not necessary here. Check this doc: https://www.pluralsight.com/guides/test-asynchronous-code-jest